CHANGELOG.md

```diff
@@ -3,19 +3,55 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0.1 - [2024-04-02]
-
-- [#277](https://github.com/nf-core/metatdenovo/pull/277) - Fix mistake in how `--eukulele_db` parameter is handled. Remove possibility to use a list of dbs in the same run.
-- [#277](https://github.com/nf-core/metatdenovo/pull/277) - Gzip user provided assembly files to avoid overwriting by assuming they're already zipped.
-
-## v1.0.0 - [date]
+## v1.1.0dev - [date]
```

Comment:
This looks funny as I just reviewed a PR where you updated to `1.1.0`. Didn't you merge the last PR into dev before you did this PR?

Reply:
Yes, I did. weird.